setup.py

```diff
@@ -100,6 +100,9 @@ def run(self):
         # We don't populate __file__ properly at the top level or something?
         # Also I think there would be problems with how we generate version.py.
         'version.py',
+
+        # Written by someone who doesn't know how to deal with mypyc
+        'stubtest.py',
```

Comment:
Is this still necessary?

Reply:
Yeah, I never figured out how to get vars / __dict__ to work. This is the big off switch I mentioned in https://github.com/python/mypy/pull/8380#issuecomment-583677692

Reply:
Makes sense, I inferred from some of the changes since that you'd figured out a workaround. I merged this change now and hopefully we'll eventually find a way to remove the special-casing.

Reply:
The workarounds were for the bug in early Python 3.5s and a valid mypyc complaint from another module.
I’ll try Sully’s suggestion to see if we can remove stubtest from the blacklist.